### PR TITLE
🚧Update all contribution docs

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
Refer to issue for more info

### CODE REVIEW

File: docs/BranchingModel.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<h1 align="center">Branching</h1>
+
+This document illustrates a clear purpose for adding features in **preview** and **production** releases.
+
+As any contributor, you only have to worry about creating <span style="color: #66B2FF;font-weight:bold">feature</span> branches from the <span style="color: #82B366;font-weight:bold">main</span> or <span style="color: #CC99FF;font-weight:bold">preview</span> branch and creating pull requests to merge those branches into the same branch that the **feature** branch was created from.  The rest is taken care of by a solid CI/CD system as well as the maintainers of the project.  Only the organization owner and designated team members will manage the release process.  So, contributing is very easy!!🥳
+
```
Comment (kselena) [inline docs/BranchingModel.md:6]: Reword 
```suggestion
As a contributor, you only create <span style="color: #66B2FF;font-weight:bold">feature</span> branches from the <span style="color: #82B366;font-weight:bold">main</span> or <span style="color: #CC99FF;font-weight:bold">preview</span> branch.  Your pull requests will merge into the same branch that the **feature** branch was created from.  The rest is taken care of by a solid CI/CD system as well as the maintainers of the project.  The organization owner and designated team members will manage the release process.  So, contributing is very easy!!🥳

```

File: docs/BranchingModel.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+A long-living branch that represents stable production versions:
+- **Branch syntax:** main
+- **Branches that can be merged into this one:**
+  - <span style="color: #66B2FF;font-weight:bold">feature</span> branches via pull request
+  - <span style="color: #CC99FF;font-weight:bold">preview</span> branches via pull request
+- **Created from:** none
+- **Merges into:** none
+- **Required casing:** all lowercase
+- **CI/CD:** Pull requests that merge into this branch contain various status checks depending on the project and all must pass before they can be merged into this branch.
```
Comment (kselena) [inline docs/BranchingModel.md:21]: Reword
```suggestion
- **CI/CD:** Pull requests that merge into this branch contain various status checks and all must pass before they can be merged into this branch.
```

File: docs/BranchingModel.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Holds minimal changes required for the stability of an upcoming **production** release.  This is the branch you create **feature** branches from and merge **feature** branches into to make a change to a **preview** version of the software.
+
+**_NOTE_:** Used for refactoring, bug fixes, and changes related to making upcoming releases more stable.  This gives users the chance to utilize the software and provide feedback before a major release.  Introducing major features outside of the changes in the upcoming release is not allowed. These kinds of changes are performed on the <span style="color: #CC99FF;font-weight:bold">preview</span> branch by using <span style="color: #66B2FF;font-weight:bold">feature</span> branches.
+- **Branch syntax:** preview
+- **Branches that can be merged into this one:** <span style="color: #66B2FF;font-weight:bold">feature</span> branches
+- **Created from:** <span style="color: #82B366;font-weight:bold">main</span> branch
+- **Merged into:** <span style="color: #82B366;font-weight:bold">main</span> branch
+- **Required casing:** all lowercase
+- **CI/CD:** All status checks must pass before merging a pull request into this branch can be completed.
```
Comment (kselena) [inline docs/BranchingModel.md:34]: Reword
```suggestion
- **CI/CD:** All status checks must pass before merging a pull request into this branch.
```

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,32 +1,30 @@
...
 
 If you have any ideas or comments on how we can improve the process, please don't hesitate to reach out to one of our project maintainers. We value your input and would love to hear from you.
 
-There are several ways you can contribute to our projects, including creating GitHub issues for any problems you encounter, submitting code contributions, and more. To ensure your contributions are properly directed, please only use the issues marked with the description _**(Outside contributors only)**_. Our project maintainers will handle the rest.
+There are several ways you can contribute to our projects, including creating GitHub issues for any problems you encounter, submitting code contributions, and more. To ensure your contributions are properly directed, please only create an issue that is marked with the description _**(Outside contributors only)**_. Our project maintainers will handle the rest.
 
 Thank you again for your interest in contributing to our projects, and we look forward to working with you!
```
Comment (kselena) [inline docs/CONTRIBUTING.md:9]: Reword?
```suggestion
Thank you again for your interest in contributing to our projects, and we look forward to collaborating with you!
```

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -1,32 +1,30 @@
 <h1 style="border:0;font-weight:bold" align="center">Contribution Guide</h1>
 
-Welcome, and thank you for considering contributing to our projects! We appreciate any help you can provide and are constantly looking for ways to make the contribution process as simple as possible. Our top priority is maintaining high software quality, and we rely on the input of talented outside contributors like you to achieve that goal.
+Welcome, and thank you for your interest in contributing to our projects! We appreciate any help you can provide and are always looking for ways to make the contribution process as simple as possible. Our top priority is maintaining high software quality, and we rely on the input of talented outside contributors like you to achieve that goal.
 
 If you have any ideas or comments on how we can improve the process, please don't hesitate to reach out to one of our project maintainers. We value your input and would love to hear from you.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:5]: Reword
```suggestion
Welcome, and thank you for your interest in contributing to our projects! Our top priority is maintaining high software quality and having a smooth contribution process.  We appreciate any help you can provide, so if you have any ideas or comments on how we can improve, please reach out to one of our project maintainers. We value your input and would love to hear from you.
```

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,32 +1,30 @@
...
+There are several ways you can contribute to our projects, including creating GitHub issues for any problems you encounter, submitting code contributions, and more. To ensure your contributions are properly directed, please only create an issue that is marked with the description _**(Outside contributors only)**_. Our project maintainers will handle the rest.
 
 Thank you again for your interest in contributing to our projects, and we look forward to working with you!
 
 **Considerations:**
 
-Only the **KinsonDigital** team will merge changes that improve the library significantly. We will not merge changes that have narrowly-defined benefits or contain breaking changes in any way. All contributions must also follow all other guidelines outlined in this document.
+Only the **KinsonDigital** team will merge changes that improve the library significantly. We will not merge changes that have narrowly-defined benefits or contain breaking changes in any way without approval. All contributions must also follow all other guidelines outlined in this document.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:13]: Use contractions
```suggestion
Only the **KinsonDigital** team will merge changes that improve the library significantly. We won't merge changes that have narrowly-defined benefits or contain breaking changes in any way without approval. All contributions must also follow all other guidelines outlined in this document.
```

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -1,32 +1,30 @@
...
 **Please:**
 
-- **DO** follow the [code of conduct](https://github.com/KinsonDigital/.github/blob/master/docs/code_of_conduct.md) on GitHub.
+- **DO** follow the [code of conduct](https://github.com/KinsonDigital/.github/blob/main/docs/code_of_conduct.md) on GitHub.
 - **DO** follow the `📃rules` on [Discord](https://discord.gg/qewu6fNgv7).
 - **DO** talk to us on our [Discord](https://discord.gg/qewu6fNgv7) server.
 - **DO** keep issue discussions around contributions focused.
 - **DO** use the comment sections in the issues and pull requests (PRs) that are relevant to said issues and PRs.  This not only keeps conversations contained in the issues and PRs, but it also sends notifications to the project maintainers and shows the conversation to the community/public.
 - **DO** leave a comment on the issue that you are interested in to let the project maintainers know that you would like to work on the issue.
 - **DON'T** start working on a PR until you get confirmation from a project maintainer and the issue is assigned to you.  We do not want to waste your time on an issue that you are not allowed to work on.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:25]: use contractions
```suggestion
- **DON'T** start working on a PR until you get confirmation from a project maintainer and the issue is assigned to you.  We don't want to waste your time on an issue that you are not allowed to work on.
```

File: docs/CONTRIBUTING.md [hunk 8]
```diff
@@ -1,32 +1,30 @@
...
 
 **Please:**
 
-- **DO** follow the [code of conduct](https://github.com/KinsonDigital/.github/blob/master/docs/code_of_conduct.md) on GitHub.
+- **DO** follow the [code of conduct](https://github.com/KinsonDigital/.github/blob/main/docs/code_of_conduct.md) on GitHub.
 - **DO** follow the `📃rules` on [Discord](https://discord.gg/qewu6fNgv7).
 - **DO** talk to us on our [Discord](https://discord.gg/qewu6fNgv7) server.
 - **DO** keep issue discussions around contributions focused.
 - **DO** use the comment sections in the issues and pull requests (PRs) that are relevant to said issues and PRs.  This not only keeps conversations contained in the issues and PRs, but it also sends notifications to the project maintainers and shows the conversation to the community/public.
 - **DO** leave a comment on the issue that you are interested in to let the project maintainers know that you would like to work on the issue.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:24]: Contractions
```suggestion
- **DO** leave a comment on the issue that you are interested in to let the project maintainers know that you'd like to work on the issue.
```

File: docs/CONTRIBUTING.md [hunk 9]
```diff
@@ -38,39 +36,27 @@ Contributions must maintain API signature and behavioral compatibility. Contribu
 
 <h2 style="border:0;font-weight:bold" align="center">Setting Things Up</h2>
 
-For first-time contributors, you need to fork the repository so you can contribute.  There are a few steps that you will need to go through to start contributing.  Go [here](https://github.com/KinsonDigital/.github/blob/master/docs/HowToFork.md) to follow the steps on forking the repository.  There are lots of resources on the web on what forking is and how to do this.
+For first-time contributors, you need to fork the repository so you can contribute.  There are a few steps that you will need to go through to start contributing.  Go [here](https://github.com/KinsonDigital/.github/blob/main/docs/HowToFork.md) to follow the steps on forking the repository.  There are lots of additional resources on the web on what forking is and how to do this.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:39]: Reword
```suggestion
For first-time contributors, you need to fork the repository to start contributing.  There are a few steps that you'll need to go through to start contributing.  Go [here](https://github.com/KinsonDigital/.github/blob/main/docs/HowToFork.md) to follow the steps on forking the repository.  There are lots of additional resources on the web on what forking is and how to do this.
```

File: docs/CONTRIBUTING.md [hunk 10]
```diff
@@ -38,39 +36,27 @@ Contributions must maintain API signature and behavioral compatibility. Contribu
...
+If the core team is unable to champion an issue, they will mark it with a "help wanted" label. This indicates that any external contributor may pick it up and implement it as part of a contribution. However, some **"help wanted"** issues may require intermediate knowledge of the codebase, area, and/or ecosystem; or may have uncertainty surrounding the implementation details.  If this is the case, talk to us in [Discord](https://discord.gg/qewu6fNgv7) or the issue's discussion. We also mark some issues with the label **"good first issue"** which indicates that an issue is straightforward and is a good place to start if you're interested in contributing but are new to the codebase.
```
Comment (kselena) [inline docs/CONTRIBUTING.md:59]: contractions
```suggestion
If the core team is unable to champion an issue, they'll mark it with a "help wanted" label. This indicates that any external contributor can pick it up and implement it as part of a contribution. However, some **"help wanted"** issues may require intermediate knowledge of the codebase, area, and/or ecosystem; or may have uncertainty surrounding the implementation details.  If this is the case, talk to us in [Discord](https://discord.gg/qewu6fNgv7) or the issue's discussion. We also mark some issues with the label **"good first issue"** which indicates that an issue is straightforward and is a good place to start if you're interested in contributing, but new to the codebase.
```

File: docs/CreatingBranches.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
+<h2 style="border:0;font-weight:bold" align="center">Create a Branch to Contribute</h2>
+
+
+When creating a branch, you need to determine if the feature or change you want to contribute is for a **production** release or a **preview** release.
+
+If you want to release something to **production**, then you create your <span style="color: #66B2FF;font-weight:bold">feature</span> branch from the <span style="color: #82B366;font-weight:bold">main</span> branch. If you want to contribute to a **preview** release, you need to create your <span style="color: #66B2FF;font-weight:bold">feature</span> branch from the <span style="color: #CC99FF;font-weight:bold">preview</span> branch.
+
+> **Note** Both branches will be merged into the same branch they were created from, via a pull request.
+
+Whatever method you want to use for creating your branches and pull requests is up to you and your preference.  But below is the git commands that you could use to help contribute to any of the **KinsonDigital** projects.
```
Comment (kselena) [inline docs/CreatingBranches.md:10]: Grammar
```suggestion
Whatever method you want to use for creating your branches and pull requests is up to you and your preference.  But below are the git commands that you can use to help contribute to any of the **KinsonDigital** projects.
```

File: docs/CreatingBranches.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+
+### **1. Sync Branch**
+
+As time moves on, the original repository changes.  This means that you need to keep your fork up to date as changes stream into the main repository.  It is important that you keep your fork up to date before you create a new branch.  Their are different options to do this.  To use the GitHub web interface go [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).  To use the GIT command line, follow the steps below.
```
Comment (kselena) [inline docs/CreatingBranches.md:15]: Reword, contractions, grammar
```suggestion
As time moves on, the original repository will change which means it's important to keep your fork up to date before you create a new branch.  There are different options to do this.  To use the GitHub web interface go [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).  To use the GIT command line, follow the steps below:
```

File: docs/HowToFork.md [hunk 13]
```diff
@@ -3,20 +3,28 @@
...
 
 ### **2. Clone the forked repository**
 
-> **Note** When cloning the repo to your machine, be sure to swap `johndoe` with your username and `myproject` with the name of the project to clone.
-   ```cli
-   git clone https://github.com/johndoe/myproject.git
-   cd myproject
-   git remote add upstream https://github.com/KinsonDigital/myproject.git
-   ```
+When setting up the cloned fork of the repository on your machine, adding a new remote of the original project is important.  It is common practice to name the new remote as `upstream`.  This remote represents the original repository that your repository was forked from.
```
Comment (kselena) [inline docs/HowToFork.md:10]: Contractions
```suggestion
When setting up the cloned fork of the repository on your machine, adding a new remote of the original project is important.  It's common practice to name the new remote as `upstream`.  This remote represents the original repository that your repository was forked from.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 4 above)
Comment (CalvinWilkinson) [reply docs/CONTRIBUTING.md:9]: Much better yes!!  😀

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Pull request merged**